pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -183,3 +85,49 @@ func (util *VsphereDiskUtil) DeleteVolume(vd *vsphereVolumeDeleter) error {
 	glog.V(2).Infof("Successfully deleted vsphere volume %s", vd.volPath)
 	return nil
 }
+
+// Checks if the specified path exists
+func pathExists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	} else if os.IsNotExist(err) {
+		return false, nil
+	} else {
+		return false, err
+	}
+}
+
+func unmountPDAndRemoveGlobalPath(globalMountPath string, mounter mount.Interface) error {
```

Comment:
Remove this method and call `UnmountPath(...)` directly in `pkg/volume/util/util.go` instead.